src/http/local_async_connection_base.hpp

```diff
@@ -1,6 +1,10 @@
 #ifndef BOOST_NETWORK_PROTOCOL_HTTP_IMPL_LOCAL_ASYNC_CONNECTION_BASE_20170307
```

Comment:
this commit only comments on 6 out of the 10 files under src/http. Please add the provenance comments to the remaining 4 files, including "src/http/traits/local_resolver.hpp"

Reply:
The files that don't have the comments aren't replicas -- they are complete reimplementations that don't share enough code with the originals. The reason these files needed to be copied is because they weren't designed to be specialized -- I'll fix that upstream in a separate refactoring.

Reply:
I see. Since all names start with local_ it sounded like they were all copies.

Reply:
Yeah. `local_` refers to "local sockets" (a.k.a., Unix domain sockets), rather than a "local copy".